snippets/with_exclusions_external_data_and_config.js

```diff
@@ -0,0 +1,38 @@
+// if you don't specify a html file, the sniper will generate a div
+var app = require("ProtVista");
+var instance = new app({
+    el: yourDiv,
+    uniprotacc : 'P05067',
+    customDataSources: [
+        {
+            url: './data/externalFeatures_',
+            authority: 'myLab',
```

Comment:
Looking at tooltips, I meant adding a description to tooltips. Is that documented?